radio/src/thirdparty/FatFs/ffconf.h

```diff
@@ -78,8 +78,10 @@
 /*---------------------------------------------------------------------------/
 / Locale and Namespace Configurations
 /---------------------------------------------------------------------------*/
-
-#define FF_CODE_PAGE	437
+#if defined(TRANSLATIONS_RU)
+#else
+#define _CODE_PAGE	437
```

Comment:
You removed define for FF_CODE_PAGE ?

Reply:
Sorry this is my mistake from the experiments